src/main/java/org/prebid/server/health/HealthChecker.java

```diff
@@ -0,0 +1,12 @@
+package org.prebid.server.health;
+
+import org.prebid.server.health.model.StatusResponse;
+
+public interface HealthChecker {
+
+    String name();
+
+    void initialize();
```

Comment:
This method should be moved to PeriodicHealthChecker class because it mainly used in pair with update status functionality.